bin/generate_jobs_yaml.rb

```diff
@@ -32,14 +32,29 @@
 nginx:1.24.0 1.24.0
 TAB
 
+resty_version_table = <<-TAB
+openresty/openresty:1.27.1.1-alpine 1.27.1.1
+openresty/openresty:1.25.3.2-alpine 1.25.3.2
+openresty/openresty:1.25.3.1-alpine 1.25.3.1
+openresty/openresty:1.21.4.4-alpine 1.21.4.4
+openresty/openresty:1.21.4.3-alpine 1.21.4.3
+openresty/openresty:1.21.4.2-alpine 1.21.4.2
+openresty/openresty:1.21.4.1-alpine 1.21.4.1
```

Comment:
These are outdated versions of NGINX. Our policy supports only the four most recent versions. Could you remove these?